spec/system/managing_topics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics' do
+  before do
+    FactoryBot.create(:topic, name: 'What is your favourite colour')
+  end
+
+  scenario 'when viewing' do
+    visit('/topics')
+    expect(page).to have_content('Topics')
+    expect(page).to have_content('What is your favourite colour')
+  end
+
+  scenario 'can navigate to new topic page' do
+    visit('/topics')
+    expect(page).to have_link('Add your own topics here',href: new_topic_path)
+  end
+
+  scenario 'when creating a topic' do
+    visit '/topics/new'
+    fill_in 'Name', with: 'tell us a story'
+    click_button 'Create'
+    expect(page).to have_content 'tell us a story'
+  end
+
+  scenario 'when forgetting to complete add my topic' do
```

Comment:
System/feature specs should test a single user operation. In this case, the user is trying to _create a topic_. so roll this spec into the the _'when creating a topic'_ spec. The clue is that "Forgetting to complete ..." is not a feature. You'll often find that when user stories are broken down to the lowest granularity, there will be one system/feature test for each user story.